src/ILLink.RoslynAnalyzer/DataFlow/LocalDataFlowVisitor.cs

```diff
@@ -170,6 +170,9 @@ public static IMethodSymbol GetPropertyMethod (IPropertyReferenceOperation opera
 
 		public override TValue VisitPropertyReference (IPropertyReferenceOperation operation, LocalDataFlowState<TValue, TValueLattice> state)
 		{
+			if (!operation.Property.Type.IsTypeInterestingForDataflow ())
```

Comment:
We should avoid introducing DAM-specific logic into this file, which is a generic dataflow visitor. This check can be done in `TrimAnalysisVisitor` instead. For property getters, they go through `HandleMethodCall` so I think they will be handled if we add a similar check to `VisitInstanceReference` and `HandleMethodCall` (when creating the `MethodReturnValue`) instead.